langlib/lang.runtime/src/main/ballerina/runtime.bal

```diff
@@ -102,6 +131,12 @@ public type StopHandler function() returns error?;
 # that was passed as an argument; the handler functions will be called
 # after calling `gracefulStop` on all registered listeners,
 # in reverse order of the corresponding calls to `onGracefulStop`.
+# 
+# ```ballerina
+# runtime:StopHandler handler = function() returns error? {};
```

Comment:
I think it is better to use `stopHandler` as the variable name instead of `handler`